packages/connect/src/api/resetDevice.ts

```diff
@@ -18,7 +17,7 @@ export default class ResetDevice extends AbstractMethod<'resetDevice', PROTO.Res
 
         const { payload } = this;
         // validate bundle type
-        Assert(ResetDeviceSchema, payload);
+        Assert(PROTO.ResetDevice, payload);
```

Comment:
This allows for "stringy" version of the BackupType. In the original implementation, allowed only for the numbers (here https://github.com/trezor/trezor-suite/pull/11907/files#diff-959b728cb5801ede26ea5d8fd39e563f6d37c4468c720ddbb98bb596006be76dL13 )

Reply:
But will the FW accept both number and string values?